velox/functions/sparksql/Register.cpp

```diff
@@ -45,6 +45,7 @@ static void workAroundRegistrationMacro(const std::string& prefix) {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_array_contains, prefix + "array_contains");
   VELOX_REGISTER_VECTOR_FUNCTION(
       udf_array_intersect, prefix + "array_intersect");
+  VELOX_REGISTER_VECTOR_FUNCTION(udf_arrays_overlap, prefix + "arrays_overlap");
```

Comment:
arrays_overlap is a Presto function, hence, should not appear in sparksql folder. Please, remove.